flask/logging.py

```diff
@@ -1,94 +1,69 @@
-# -*- coding: utf-8 -*-
-"""
-    flask.logging
-    ~~~~~~~~~~~~~
-
-    Implements the logging support for Flask.
-
-    :copyright: (c) 2015 by Armin Ronacher.
-    :license: BSD, see LICENSE for more details.
-"""
-
 from __future__ import absolute_import
 
+import logging
 import sys
 
 from werkzeug.local import LocalProxy
-from logging import getLogger, StreamHandler, Formatter, getLoggerClass, \
-     DEBUG, ERROR
-from .globals import _request_ctx_stack
-
 
-PROD_LOG_FORMAT = '[%(asctime)s] %(levelname)s in %(module)s: %(message)s'
-DEBUG_LOG_FORMAT = (
-    '-' * 80 + '\n' +
-    '%(levelname)s in %(module)s [%(pathname)s:%(lineno)d]:\n' +
-    '%(message)s\n' +
-    '-' * 80
-)
+from .globals import request
 
 
 @LocalProxy
-def _proxy_stream():
-    """Finds the most appropriate error stream for the application.  If a
-    WSGI request is in flight we log to wsgi.errors, otherwise this resolves
-    to sys.stderr.
+def wsgi_errors_stream():
+    """Find the most appropriate error stream for the application. If a request
+    is active, log to ``wsgi.errors``, otherwise use ``sys.stderr``.
+
+    If you configure your own :class:`logging.StreamHandler`, you may want to
+    use this for the stream. If you are using file or dict configuration and
+    can't import this directly, you can refer to it as
+    ``ext://flask.logging.wsgi_errors_stream``.
```

Comment:
Naive reader feedback: I don't know what this format is, and it might be good to elaborate on how to use this seemingly-a-URI. Can this be passed to `logging.getLogger` and it'll evaluate it lazily?